core/network/src/protocol/sync.rs

```diff
@@ -198,7 +198,9 @@ pub struct Status<B: BlockT> {
 	/// Target sync block number.
 	pub best_seen_block: Option<NumberFor<B>>,
 	/// Number of peers participating in syncing.
-	pub num_peers: u32
+	pub num_peers: u32,
```

Comment:
Why are these values `u32` and not `usize`?

Reply:
Because they don't depend on machine pointer size. `u32` is more than enough.